src/plots/linePlot.ts

```diff
@@ -424,46 +424,14 @@ export class Line<X> extends XYPlot<X, number> {
   }
 
   public entitiesAt(point: Point): IPlotEntity[] {
-    const entity = this.entityNearestByXThenY(point);
+    const entity = this.entityNearest(point);
     if (entity != null) {
       return [entity];
     } else {
       return [];
     }
   }
 
-  /**
```

Comment:
Maybe we shouldn't delete this because technically this would be a public API break and require a major version bump